examples/movies/server/src/movies/package.scala

```diff
@@ -98,7 +102,14 @@ package object movies {
       implicit val mapper = if (req.pretty) prettyJson else textJson
       val rsp = req.response
       rsp.status = Status.Ok
-      rsp.etag = time
+      //rsp.etag = time
+      //Added
+	  rsp.date = req.lastModificationBefore
```

Comment:
Oh, I missed this. Date and Last-Modified need to be formatted like 

    Mon, 23 Jun 22:23:44 2014 GMT

You can look at Joda-Time and figure out how to get that formatting.